Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -112,6 +112,7 @@ export class AppComponent implements OnInit {
         height: '90vh',
         width: '90vw'
       });
+      this.releaseMigrationService.newUpdate = true;
```

Comment:
is this for debugging purposes, or anything else?

Reply:
@DanielHabenicht I think that if the application has started for the first time the update-badge should also shown, or not? I'am also not sure^^

Reply:
Well but this case should be covered by the normal update procedure as the default value for version is `0.0.0`